bpf/lib/nodeport.h

```diff
@@ -589,6 +589,8 @@ int tail_nodeport_nat_ingress_ipv6(struct __ctx_buff *ctx)
 
 	ret = snat_v6_rev_nat(ctx, &target);
 	if (IS_ERR(ret)) {
+		if (ret == DROP_INVALID)
+			goto drop_err;
```

Comment:
AFAIU, and I think it's in line with Martynas's comment, there are several error cases where it's unnecessary to recircle. Shouldn't we instead check the case where we do need to perform the tail call? That would seem more coherent with the code comment just below your changes.

Reply:
Hello @brb @pchaigno , How do you feel with the last changes? anychance that you trigger CI to ensure everything is working right?

Reply:
Code changes look good to me.